website/content/docs/configuration/client.mdx

```diff
@@ -247,15 +247,17 @@ chroot as doing so would cause infinite recursion.
   access files only within the [task working directory].
 
 - `max_stale` `(string: "87600h")` - This is the maximum interval to allow "stale"
-  data. By default, only the Consul leader will respond to queries. Requests to
-  a follower will forward to the leader. In large clusters with many requests,
-  this is not as scalable. This option allows any follower to respond to a query,
-  so long as the last-replicated data is within this bound. Higher values result
-  in less cluster load, but are more likely to have outdated data.
+  data. If `max_stale` is not set, only the Consul leader will respond to queries, and
```

Comment:
```suggestion
  data. If `max_stale` is set to `0`, only the Consul leader will respond to queries, and
```

Since the default is 87.6k hours, users have to set it to 0 explicitly to get this behavior. It may be "not set" from Nomad's perspective of 0 values, but it **is set** from the user's perspective of having to write `max_stale = "0"` in their configuration file.

Reply:
Nice distinction. Thank you.